mkdocs.yml

```diff
@@ -1,4 +1,90 @@
-site_name: autobot Docs
+# AutoBot Information
+site_name: AutoBot Documentation
 
+# Configuration
 theme:
-  name: "material"
\ No newline at end of file
+  name: material
+  features:
+    - navigation.tabs
+    - navigation.sections
+    - toc.integrate
+    - navigation.top
+    - header.autohide #the header is automatically hidden when the user scrolls past a certain threshold, leaving more space for content. Add the following lines to mkdocs.yml:
+    - search.suggest
+    - search.share
+    - search.highlight
+  language: en
+  palette:
+    - scheme: default
+      toggle:
+        icon: material/toggle-switch-off-outline 
+        name: Switch to dark mode
```

Comment:
thank you @moses-kisakye for the contribution, am super fascinated by the dark mode feature :handshake:  :blush: